packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts

```diff
@@ -170,6 +174,39 @@ export function createMikrORMEntity() {
       return
     }
 
+    /**
+     * Defining an id property
+     */
+    if (field.dataType.name === "id") {
+      const IdDecorator = field.dataType.options?.primaryKey
```

Comment:
q: would it have the same effect to use the property decorator with primary: true?

Reply:
No idea. I decided to stick with what rest of the codebase is using.

Reply:
Lets go with that and de can revisit this later it is not 😅